projects/ngx-avatars/package.json

```
@@ -27,7 +27,7 @@
     "tslib": "^2.6.3"
   },
   "peerDependencies": {
-    "@angular/common": "^18.0.1",
-    "@angular/core": "^18.0.1"
+    "@angular/common": "^19.0.0",
+    "@angular/core": "^19.0.0"
```

Could we future proof angular peerDependencies?
```suggestion
    "@angular/common": ">=19.0.0",
    "@angular/core": ">=19.0.0"
```